### DISCUSSION TITLE
Failed to run task: 'yarn install --pure-lockfile'

### DISCUSSION THREAD

**Discussion by Azinale** [dcm4che/dcm4chee-arc-light#4527] [Q&A] [answered]
Hi, i've got this issue in dcm4chee-arc-ui2 whille running install. What should i do to fixit?
![image](https://github.com/dcm4che/dcm4chee-arc-light/assets/102240290/13847b80-26b3-4dbc-a743-6e18280ea6ad)
thanks a lot.

**Comment by vrindanayak** [comment, marked as answer]
Please retry the build. It is currently not reproducible.
